environments/staging/app-processes.yml

```diff
@@ -2,7 +2,8 @@ celery_processes:
   None:
     # celery_periodic is intentionally absent from here,
     # so that no SMSes, etc. ever come from staging.
-    celery_periodic:
+    celery_periodic: {}
+    beat: {}
```

Comment:
Does this add celery beat to staging? As the comment above says, we don't want that. If this is a mistake (and not just me misunderstanding the semantics of this new `beat` "queue"), then this is the kind of mistake that having the `server_whitelist` check was meant to prevent.

Reply:
from my understanding, that is adding `beat` to the server `None` so it is just being explicit that we have decided not to run those queues. let me know if my understanding is mistaken

Reply:
Oh, sorry, I missed the none above, thanks